src/ui/dialogs/scenesettingsdialog.cpp

```diff
@@ -186,6 +186,7 @@ SceneSettingsDialog::SceneSettingsDialog(const QString &name,
     connect(mCancelButton, &QPushButton::released,
             this, &SceneSettingsDialog::reject);
     connect(this, &QDialog::rejected, this, &QDialog::close);
+    connect(mTypeTime, QOverload<int>::of(&QComboBox::currentIndexChanged),this, [this]() { updateDuration(); });
```

Comment:
Since you have added a slot (updateDuration) I think this should be a normal signal/slot connection.

Reply:
OK, I think I have it